oak-parent/pom.xml

```diff
@@ -111,25 +110,17 @@
       <plugins>
         <plugin>
           <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.8.1</version>
           <configuration>
+            <target>${java.version}</target>
+            <source>${java.version}</source>
+            <release>${java.version}</release>
```

Comment:
Rather overwrite `maven.compiler.target` and set `maven.compiler.release` as well to the same value, as that also affects m-javadoc-p, ...

Reply:
Please add to the PR...